industry_benchmarks/input_structures/prepared_structures/jacs_set/thrombin/protein.pdb

```diff
@@ -0,0 +1,5799 @@
+TITLE     throm 2zff, modelled loop, no ZOB, xtal waters, no water,
+TITLE    2 HIP 57
+REMARK   4      COMPLIES WITH FORMAT V. 3.0, 1-DEC-2006
+REMARK 888
+REMARK 888 WRITTEN BY MAESTRO (A PRODUCT OF SCHRODINGER, LLC)
+ATOM      1  CH3 ACE L   0      17.735  17.244  20.304  1.00  0.00           C  
+ATOM      2  C   ACE L   0      16.854  16.387  19.390  1.00  0.00           C  
+ATOM      3  O   ACE L   0      17.162  15.214  19.286  1.00  0.00           O  
+ATOM      4 1H   ACE L   0      18.437  17.827  19.707  1.00  0.00           H  
+ATOM      5 2H   ACE L   0      17.136  17.911  20.923  1.00  0.00           H  
+ATOM      6 3H   ACE L   0      18.318  16.606  20.968  1.00  0.00           H  
+ATOM      7  N   CYS L   1      10.980  17.261  19.436  1.00  0.00           N  
```

Comment:
Just making a comment that the PDB ordering here has changed (compared to the Schrodinger input), but it looks lilke it's running fine.